src/config/config.json

```diff
@@ -1 +1,11 @@
-{}
\ No newline at end of file
+{
+  "GATEWAY_BASE_URL": "http://localhost:3000/v1/60676c9f7cbbfe002f0b9cbe",
+  "HTTP_CONFIG": {
+    "timeout": 0,
+    "headers": {
+      "Content-Type": "application/json",
+      "Blockchain-Subdomain": "mainnet"
+    }
+  },
+  "CHAIN_ID": "mainnet"
+}
```

Comment:
Both suggestions are favorable. Why is this a `.json` and not a bunch of ENVs is simply because I did not really give it any thought and that was what already was in there.

I'm going for env, thanks!

Reply:
Reiterating, won't go the env route just because ci has a step that writes out to this file and it is more costly to do env as I'll have to integrate the `dotenv` module.

Reply:
Even though adding the `dotenv` module and changing the CI can take some time, I will strongly recommend to do it at this point.

Reply:
Great guys I appreciate your eyes on this, some mofongo on me some time soon-ish!